README.md

```diff
@@ -230,9 +233,13 @@ yaml_files = [Path("/path/to/training_yml.yml)]
 # where to save the output
 output_directory = Path("/path/to/saved_training_data")
 
+home = Path.home()
+install_path = home / ".cellfinder"  # default
+
 run_training(
     output_directory,
     yaml_files,
+    install_path=install_path,
     learning_rate=0.0001,
```

Comment:
```suggestion
    learning_rate=1e-4,
```

Reply:
I haven't touched this line, so I'm going to leave this